internal/cmd/invoice_list.go

```diff
@@ -23,7 +24,15 @@ var listInvoicesCmd = &cobra.Command{
 			return err
 		}
 
-		invoices, err := client.Invoices.List()
+		if invoiceType == "" {
+			invoiceType = "issued"
```

Comment:
You can set `"issued"` as the default value for the flag on its configuration instead of `""`